dev-cpp/gsl/gsl-0_pre20171105.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="Guideline Support Library implementation by Microsoft."
+HOMEPAGE="https://github.com/Microsoft/GSL"
+SRC_URI="https://github.com/Microsoft/GSL/archive/master.zip -> ${P}.zip"
```

Comment:
as discussed on IRC and suggested by @benkohler `SRC_URI="https://github.com/something/${PN}/tarball/${COMMIT} -> ${P}.tar.gz"`